grisera_api/scenario/scenario_model.py

```diff
@@ -21,10 +21,10 @@ class ScenarioOut(ScenarioIn):
     Model of scenario to send to client as a result of request
 
     Attributes:
-        id (Optional[int]): Id of scenario returned from graph api
+        activities (List[ActivityOut]): List of activities in scenario
```

Comment:
There doesn't exist something like scenario in database. That terminal is only to connect newly created activities and experiment. So i don't think ID should be here, since we don't get it from database and scenario is not an object which we create directly.

Reply:
Good to know 😀